pontos/release/release.py

```diff
@@ -199,197 +194,242 @@ def commit_files(
     )
 
 
+def upload_assets(
+    username: str,
+    token: str,
+    pathnames: List[str],
+    github_json: Dict,
+    path: Path,
+    requests_module: request,
```

Comment:
```suggestion
    requests_module: requests,
```

Use the other import ...